app/src/pages/inside/profilePage/profilePage.scss

```diff
@@ -29,12 +29,16 @@
     .section-wrapper {
       padding: 0 30px;
     }
+    [class^="navigationTabs__navigation-tabs"] {
```

Comment:
I think that we can achieve the same result by adding NavigationTabs to the section-wrapper div
<img width="771" alt="Screenshot 2023-10-03 at 10 56 42" src="https://github.com/reportportal/service-ui/assets/84175555/ce527a2b-ea3a-499d-9ecb-329513f96725">
And we do not change anything in CSS

Reply:
Done.